docs/metrics/work-in-progress.md

```diff
@@ -0,0 +1,20 @@
+# Work in progress metrics
+
+## Work in progress
+
+To provide you with an high-level view of the current work in progress of your teams, Pulse groups pull requests that are currently open by the following phases:
+
+-   **Open:** the pull request is open but the team hasn't interacted or reviewed it.
+-   **In review:** the pull request received at least one review but it isn't approved yet. Reviews include approvals, change requests, and inline pull request comments, but don't include conversation comments.
+-   **Approved:** the pull request received at least one approval.
+
+## Work in progress details
+
+The following metrics for each open pull request allow you to understand which work items are about to become or have become blockers and may need an action to move forward:
+
+-   **Time in progress:** number of days that the pull request is in progress.
```

Comment:
```suggestion
-   **Time in progress:** time since that pull request is in progress.
```
The time unit depends on the worse time of the chart. It can be any unit

Reply:
Thanks, it makes sense.